tests/Oryx.BuildImage.Tests/Java/JavaDynamicInstallationTest.cs

```diff
@@ -0,0 +1,72 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Oryx.BuildScriptGenerator.Common;
+using Microsoft.Oryx.BuildScriptGenerator.Java;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.BuildImage.Tests
```

Comment:
I couldn't find where these tests are executed in the pipeline. Looking at the two "Publish test results" steps for both Build and Runtime, I couldn't find them (or the previously existing tests for that matter). Do you know where they are executed?

Reply:
Yes, they're executed, the total number of tests is 372(scroll down at the bottom of build image tests), compared to previously we had 367.

Reply:
Thanks!